test/test_list.sh

```diff
@@ -23,7 +23,7 @@ for v in 0.7.2 0.7.13 0.9.1 0.9.2 0.9.11; do
   tfenv install ${v} || error_and_proceed "Install of version ${v} failed"
 done
 
-result=$(tfenv list)
+result=$(tfenv list | sed 's/ //g;s/^*//;s/(.*)//')
```

Comment:
I think it's better to check the output of `tfenv list` as is rather than to remove decorations from it. I mean `expected` should be 

```
* 0.9.11 (set by ${TFENV_ROOT}/versions)
  0.9.2
  0.9.1
  0.7.13
  0.7.2
```

Reply:
You're right, it's more coherent